stl/inc/ostream

```diff
@@ -912,15 +912,45 @@ basic_ostream<char, _Traits>& operator<<(
 }
 
 #ifdef __cpp_char8_t // These deleted overloads are specified in P1423.
+// don't insert a UTF-8 NTBS
 template <class _Traits>
-basic_ostream<char, _Traits>& operator<<(
-    basic_ostream<char, _Traits>&, const char8_t*) = delete; // don't insert a UTF-8 NTBS
+basic_ostream<char, _Traits>& operator<<(basic_ostream<char, _Traits>&, const char8_t*) = delete;
+template <class _Traits>
+basic_ostream<wchar_t, _Traits>& operator<<(basic_ostream<wchar_t, _Traits>&, const char8_t*) = delete;
 
+// don't insert a UTF-8 code unit
 template <class _Traits>
-basic_ostream<char, _Traits>& operator<<(
-    basic_ostream<char, _Traits>&, char8_t) = delete; // don't insert a UTF-8 code unit
+basic_ostream<char, _Traits>& operator<<(basic_ostream<char, _Traits>&, char8_t) = delete;
+template <class _Traits>
+basic_ostream<wchar_t, _Traits>& operator<<(basic_ostream<wchar_t, _Traits>&, char8_t) = delete;
 #endif // __cpp_char8_t
 
+#if !_HAS_STREAM_INSERTIONS_REMOVED_IN_CXX20

```

Comment:
I believe that is a wrong negation`. If `_HAS_STREAM_INSERTIONS_REMOVED_IN_CXX20` is set they should be removed

Reply:
My mental expansion of the macro is "[The STL] has [the] stream insertions [which are] removed in c++20," for which the current logic is correct, as opposed to "[The STL] has stream insertions removed [as specified] in C++20." The fact that both interpretations work suggests this is a bad name; I note that `_ENABLE_BOGUS_STREAM_INSERTIONS` was unambiguous ;)

Reply:
Maybe following the naming in #380 
```cpp
#define _HAS_REMOVED_STREAM_INSERTIONS
```

Or maybe even more correct
```cpp
#define _HAS_DELETED_STREAM_INSERTIONS
```

Reply:
I'm concerned that "stream insertion [operator]s" is a bit too generic of a feature name, and that we may see more in the future which would make this name ambiguous. I think  `_HAS_CXX20_DELETED_STREAM_INSERTIONS` would therefore be a better choice.

Reply:
Using #380's `_HAS_FEATURES_REMOVED_IN_CXX20` as precedent, how about `_HAS_STREAM_INSERTION_OPERATORS_DELETED_IN_CXX20`? I believe this avoids ambiguity. I think I also prefer to talk about "insertion operators". (Compare "assignments" which refers to actually performing work, versus "assignment operators" in source code.)

Reply:
The internal PR passed testing with `_HAS_STREAM_INSERTIONS_REMOVED_IN_CXX20`, it's now undergoing another round after renaming to `_HAS_STREAM_INSERTION_OPERATORS_DELETED_IN_CXX20`. I'm happy to merge everything after that test round completes, so AFAIK we're only blocked on @miscco's signoff on the name `_HAS_STREAM_INSERTION_OPERATORS_DELETED_IN_CXX20`.